### PR TITLE
Microverse pull request

### PR DESCRIPTION
**Pull request by clin2on3mun**
1.Set up linters
2. Added HTML and CSS file
3. Added test.md file and added it to gitignore
4. Added a template README.md file

### CODE REVIEW

**Comment by clin2on3mun** [comment]
simple html and css project

**Comment by clin2on3mun** [comment]
the simple html and css project

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>hello microverse</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>Hello Microverse</h1>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you give your `h1` tag a class name and the name given to it should be used to style the content in your `CSS` file as required in the project requirement. 👍 👍

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @clin2on3mun, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x] I would kindly suggest you make your Pull Request title to be more descriptive to denote what the project you worked on is all about and also give it summary of what the project entails. Kindly work on it, you can use this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for a better understanding. 👍 👍

- [x] Kindly make your README more descriptive and professional by adding the table of content to it as seen in the template, 
  - Add the Future Feature section to it and include 2-3 aspects of the project that you will add in the future
  - Kindly add the MIT license section to it and be sure the Link works perfectly by taking us to the licensed copyright you add in your project root directory.

Here is the new README template given to us by MIcroverse: [LINK](https://github.com/microverseinc/readme-template)

-N.B: Kindly remove the Microverse Logo at the top of the README file, optionally you can add your logo if you have one

- [x] I would suggest you update your `Html-and-css` branch by pulling your changes from the main branch to the current branch `i.e Html-and-css`

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,8 +0,0 @@
-The project title is Hello microverse
-Tech Stacks:
-    -Html
-    -Css
-Author:
- Name:Munana Clinton
- twitter:clintonmunana@gmail.com
- Github:https://github.com/clin2on3mun
```
Comment (roshan-bajgain) [inline README.md]: - [x] As requested by the previous reviewer kindly update the README file, please use your project name here and also remove the 2nd author as there is no 2nd author also your MIT license is broken please fix it
### Place to replace
Annotation 2023-01-11 165220
Annotation 2023-01-11 165133

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @clin2on3mun, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
- [ ] You have a branch conflict please fix it 
### Conflict


Annotation 2023-01-11 165655
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AHP15 — COMMENTED** [review]
Hi @clin2on3mun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ No Linter errors
✅ Good GitHub Flow 
## Optional suggestions
## Note: Please next time do not merge your branch until it get approved 😉 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — COMMENTED** [review]
Hi @clin2on3mun 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with a wrong link, kindly submit the right link, we can't wait to see your work😉🤗

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**